lib/datasets/ita-corpus.rb

```diff
@@ -0,0 +1,72 @@
+require 'csv'
+
+require_relative 'dataset'
+
+module Datasets
+  class ITAcorpus < Dataset
```

Comment:
Could you use CamelCase for class name?

```suggestion
  class ITACorpus < Dataset
```

Reply:
I'll remember this. Thank you.